pkgs/applications/misc/hollywood/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, makeWrapper, lib, coreutils, apg, atop, bmon, cmatrix
+, pygments, moreutils, util-linux, jp2a, man, mplayer, openssh, tree, mlocate
+, findutils, ccze, ncurses, python3, wget, libcaca, newsboat, rsstail, w3m
+, ticker, tmux, byobu }:
+let
+  pathDeps = [
+    tmux
+    coreutils
+    ncurses
+    jp2a
+    mlocate
+    apg
+    atop
+    bmon
+    cmatrix
+    pygments
+    moreutils
+    util-linux
+    jp2a
+    man
+    mplayer
+    openssh
+    tree
+    findutils
+    ccze
+  ];
+in stdenv.mkDerivation {
+  pname = "hollywood";
+  version = "1.22";
+  src = fetchFromGitHub {
+    owner = "dustinkirkland";
+    repo = "hollywood";
+    rev = "35275a68c37bbc39d8b2b0e4664a0c2f5451e5f6";
+    sha256 = "sha256-faIm1uXERvIDZ6SK6uarVkWGNJskAroHgq5Cg7nUZc4=";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+  patches = [
+    ./nixos-paths.patch
+    ./remove-speedometer.patch
+    ./remove-wallstreet.patch
+  ];
+  buildPhase = "";
```

Comment:
If there is no building phase, use `dontBuild = true;` instead.

Reply:
Done